src/app/money.pipe.ts

```diff
@@ -0,0 +1,27 @@
+import {Pipe, PipeTransform} from "@angular/core";
+import {Money} from "./mandate.model";
+import {ExactCurrencyPipe} from "./exact-currency.pipe";
+
+/**
+ * Formats a money object as a string. Currently, the money object is only used in regular giving mandates, (hence
+ * the type being defined in the mandate model file) but it may be expanded to much wider use in future).
+ */
+@Pipe({
+  standalone: true,
+  name: 'money',
+})
+export class MoneyPipe implements PipeTransform {
+  transform(money: Money): string {
```

Comment:
angular is clever enough to read this and stop us using this pipe with any value that isn't statically typed as `Money`.